pages.ta/common/java.md

```diff
@@ -0,0 +1,24 @@
+# java
+
+> ஜாவா பயன்பாட்டு துவக்கி.
+> மேலும் தகவலுக்கு: <https://java.com>.
+
+- ஒரு main செயல்பாட்டைக் கொண்ட ஜாவா .class கோப்பை வெறும் class பெயரை பயன்படுத்தி இயக்கவும்:
+
+`java {{classname}}`
+
+- ஒரு .jar நிரலை இயக்கவும்:
+
+`java -jar {{filename.jar}}`
```

Comment:
Can we translate `filename` here at all? The filename extension `.jar` needs to stay the same, but I assume that `filename` could still be translated? Same goes for below too.

Reply:
@sbrl added translations for the parameters.